src/plugins/workspace/public/components/workspace_form/use_workspace_form.ts

```diff
@@ -149,6 +148,9 @@ export const useWorkspaceForm = ({
       },
       false
     );
+    if (result?.success) {
```

Comment:
Shall we show something if it fails?

Reply:
We will toast if fails. This is just for onChanging state only when API success.